src/firebase/auth.js

```diff
@@ -5,13 +5,13 @@ import { getFirestore } from "firebase/firestore";
 import {getAnalytics} from "firebase/analytics";
 
 const firebaseConfig = {
-  apiKey: process.env.REACT_APP_apiKey,
-  authDomain: process.env.REACT_APP_authDomain,
-  projectId: process.env.REACT_APP_projectId,
-  storageBucket: process.env.REACT_APP_storageBucket,
-  messagingSenderId: process.env.REACT_APP_messagingSenderId,
-  appId: process.env.REACT_APP_appId,
-  measurementId: process.env.REACT_APP_measurementId,
+  apiKey: process.env.REACT_APP_APIKEYURL,
```

Comment:
do not change all the names. other people working on this repo will have to face some issues because of it.

Reply:
Yeah sorry for that.